src/components/headersEditor/HeadersSection.tsx

```diff
@@ -0,0 +1,22 @@
+import { Col, Row } from 'antd';
+import React, { useState } from 'react';
+
+import HeadersEditor from './HeadersEditor';
+import UrlPreview from './UrlPreview';
+
+export default function HeadersSection() {
+  const [headers, setHeaders] = useState<Record<string, string>>({});
+
+  return (
+    <>
+      <Row gutter={[16, 24]}>
+        <Col span={24}>
+          <HeadersEditor onChange={setHeaders} />
```

Comment:
```suggestion
          <HeadersEditor setHeaders={setHeaders} />
```
мб переименовать проп в `setHeaders` ? а то щас трудно понять или это какое-то конвенциональное решение?

Reply:
да, так менее абстрактно, переименовала ✅